### ISSUE TITLE
Improved Prefabs

### ISSUE THREAD

**Issue by Konclan** [ChaosInitiative/Portal-2-Community-Edition#87]
Redo most default prefabs provided by Portal 2, most of them are poorly constructed and might not even be todays mapping standard for making the prefab. This would help new mappers immensely. Also add default element prefabs or item in Portal 2 if its not provided already (There might not be any).

**Comment by SpyceTewan** [comment]
Do you mean instances?

**Comment by Konclan** [comment]
Yeah.. I’ve been modding a lot for world at war recently, radiant calls them prefabs lol

**Comment by SpyceTewan** [comment]
I can provide all of the instances I've remade for Portal: Revolution, which I'd probably also use for the P2CE maps

**Comment by donsomeone** [comment]
What's wrong with the existing ones? Just curious.

**Comment by Konclan** [comment]
They’re unoptimized, sometimes badly textured, and sometimes just the wrong way to do said thing. Also better instances usually combine different types with the use of instance kvps.

**Comment by SpyceTewan** [comment]
Created the instance repo. Closing this now
